docs/core-concepts/app-deployment.md

```diff
@@ -1,17 +1,255 @@
-# 🔧 .NET Core Application Deployment concepts
-
-> **Note**
-> 
-> This topic hasn’t been written yet! 
->
-> We welcome your input to help shape the scope and approach. You can
-> track the status and provide input on this
-> [issue](https://github.com/dotnet/core-docs/issues/471) at GitHub.
-> 
-> If you would like to review early drafts and outlines of this
-> topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on
-> [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
-	
\ No newline at end of file
+# .NET Core Application Deployment #
+
+You can deploy your .NET Core app in either of two ways:
+
+- As a portable app. A portable app relies on a shared system-wide version of the .NET Core framework that is present on a system.
+ 
+- As a self-contained application. All components, including .NET Core itself, are included with the application and are isolated from other .NET Core applications. 
+
+# Portable Applications #
+
+For a portable app, you deploy only your app and any third-party dependencies. You do not have to deploy .NET Core, since your app will use the version of .NET Core that's present on the target system. This is the default deployment model for .NET Core apps.
+
+## Why Deploy a Portable App? ##
+
+Deploying a portable app has a number of advantages:
+
+- You do not have to define the target operating systems that your .NET Core app will run on in advance. Because .NET Core uses a common PE file format for executables and libraries regardless of operating system, .NET Core can execute your app regardless of the underlying operating system.
+
+- The size of your deployment package is small. You only have to deploy your app and its dependencies, not .NET Core itself.
+
+- Multiple apps use the same .NET Core installation, which reduces both disk space and memory usage on host systems.
+
+There are also a few disadvantages:
+
+- Your app can run only if the version of .NET Core that you target, or a later version, is already installed on the host system. 
+
+- It is possible for the .NET Core runtime and libraries to change without your knowledge. In rare cases, this may change the behavior of your app.
+
+## Deploying a Simple Portable App ##
+
+Deploying a portable app with no third-party dependencies simply involves building, testing, and publishing the app. 
+A simple example written in C# illustrates the process. The example uses the dotnet utility from the command line; however, 

```

Comment:
add a relative link to the dotnet topic?
https://dotnet.github.io/docs/core-concepts/core-sdk/cli/dotnet.html